### PR TITLE
Add a gitlab job for snapshot-rebuild

### PR DESCRIPTION
**Pull request by shashank855**
Signed-off-by: ranjanshashank <ranjanshashank855@gmail.com>
**Here is the link of the pipeline:**
- https://gitlab.openebs.ci/openebs/e2e-gcp/pipelines/281

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ mongo-jiva-run/load/check 1:2:
   script: 
    - ./script/apps/mongo/workload/jiva/mongo-app-workload-jiva
 
-# POSTGRES-JIVA
+POSTGRES-JIVA
```
Comment (nsathyaseelan) [inline .gitlab-ci.yml]: @shashank855  this is a comment. Can you do comment these lines again.

Comment (shashank855) [reply .gitlab-ci.yml]: @nsathyaseelan I have made the changes

**Review by nsathyaseelan — APPROVED** [review]
LGTM

**Review by dargasudarshan — APPROVED** [review]
LGTM

**Pull request merged**